test/InteractiveTests.h

```diff
@@ -60,6 +61,7 @@ Testsuite const g_interactiveTestsuites[] = {
 	{"Yul Object Compiler", "libyul",      "objectCompiler",      false, false, &yul::test::ObjectCompilerTest::create},
 	{"Function Side Effects","libyul",     "functionSideEffects", false, false, &yul::test::FunctionSideEffects::create},
 	{"Yul Syntax",          "libyul",      "yulSyntaxTests",      false, false, &yul::test::SyntaxTest::create},
+	{"EVM Code Transform",  "libyul",      "evmCodeTransform",    false, false, &yul::test::EVMCodeTransformTest::create, {"nooptions"}},
```

Comment:
@chriseth I think the ``nooptions`` part here will have these only run once instead of with each optimizer and evm version combination.

Reply:
ah yes!